From 242823400c5bd59960e1b40d941e177e8ebad57e Mon Sep 17 00:00:00 2001
From: Ye Li <ye.li@nxp.com>
Date: Wed, 13 Dec 2017 00:22:09 -0600
Subject: [PATCH] MLK-17200-3 mx7ulp: Select the SCG1 APLL PFD as a system
 clock source

Due to the APLL out glitch issue TKT332232, the APLLCFG PLLS bit must be set
to select SCG1 APLL PFD for generating system clock to align with the design.

Signed-off-by: Ye Li <ye.li@nxp.com>
Acked-by: Peng Fan <peng.fan@nxp.com>
---
 board/freescale/mx7ulp_evk/imximage.cfg | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/board/freescale/mx7ulp_evk/imximage.cfg b/board/freescale/mx7ulp_evk/imximage.cfg
index a5c8c4e..ab7c8fe 100644
--- a/board/freescale/mx7ulp_evk/imximage.cfg
+++ b/board/freescale/mx7ulp_evk/imximage.cfg
@@ -46,7 +46,7 @@ DATA 4   0x403f00dc 0x00000000
 DATA 4   0x403e0040 0x01000020
 DATA 4   0x403e0500 0x01000000
 DATA 4   0x403e050c 0x80808080
-DATA 4   0x403e0508 0x00160000
+DATA 4   0x403e0508 0x00160002
 DATA 4   0x403E0510 0x00000002
 DATA 4   0x403E0514 0x00000005
 DATA 4   0x403e0500 0x00000001
-- 
2.9.2

